src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -64,6 +68,8 @@ export class CalciteFlowItem {
 
   @Element() el: HTMLElement;
 
+  @State() rtl: boolean = getElementDir(this.el) === "rtl";
```

Comment:
maybe every component that needs rtl support should have a state like this? Should this be updated on every render or just during initialization?